index.html

```diff
@@ -7,6 +7,7 @@
 <!--<![endif]-->
 
 <head>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-167504505-2"></script>
```

Comment:
Nice! It looks like you included the google analytics tag with the user specific id in the correct place in the head of the document.